src/Main/UserInterface.cpp

```diff
@@ -1121,7 +1149,7 @@ namespace VeraCrypt
 					" Delete keyfiles from security tokens. See also command --list-token-keyfiles.\n"
 					"\n"
 					"--export-token-keyfile\n"
-					" Export a keyfile from a security token. See also command --list-token-keyfiles.\n"
+					" Export a keyfile from a token keyfile. See also command --list-token-keyfiles.\n"
```

Comment:
there is an extra "keyfile" at the end of the first sentece. It should read "Export a keyfile from a token."